components/hook/hash-change.js

```diff
@@ -0,0 +1,21 @@
+import { useEffect } from 'react'
+
+const getHash = () => window.location.hash
+
+let currentHash
+
+export default fn => {
+  useEffect(() => {
+    window.document.addEventListener(
+      'scrollStart',
```

Comment:
This event listener approximation is just a temporal thing until determinate a better way using `Router` provided my Next.js